src/wiktextract/extractor/de/linkage.py

```diff
@@ -8,23 +8,11 @@
 from wiktextract.page import clean_node
 from wiktextract.wxr_context import WiktextractContext
 
-SEMANTIC_RELATIONS = {
-    "Gegenwörter": "antonyms",
-    "Holonyme": "holonyms",
-    "Oberbegriffe": "hypernyms",
-    "Redewendungen": "expressions",
-    "Sinnverwandte Wörter": "coordinate_terms",
-    "Sprichwörter": "proverbs",
-    "Synonyme": "synonyms",
-    "Unterbegriffe": "hyponyms",
-    "Wortbildungen": "derived",
-}
 
-
-def extract_semantic_relations(
+def extract_linkages(
     wxr: WiktextractContext, page_data: List[Dict], level_node: LevelNode
 ):
-    relation_key = SEMANTIC_RELATIONS.get(level_node.largs[0][0])
+    linkage_type = wxr.config.LINKAGE_SUBTITLES[level_node.largs[0][0]]
```

Comment:
Unless `LINKAGE_SUBTITLES` contains all the possible subtitles, this line should use `.get()`.

Reply:
Since this code is only executed after checking that the linkage_type is in `LINKAGE_SUBTITLES`, it's save. 

I am still implementing your suggestion to avoid future troubles.